.github/vale/config/vocabularies/Docker/accept.txt

```diff
@@ -35,6 +35,7 @@ Ddosify
 Debootstrap
 Dev Environments?
 Django
+docker/
```

Comment:
Hope this one works; looks like this file accepts some RegEx? Not sure how it matches things (so if it would need something like `[^]]+` after it?

Reply:
Do we need some exception for `docker-XXXX` as well? (thinking `docker-compose`)?

Reply:
I think the only valid use for `docker-compose` would be in a code span, when referring to the legacy command, so in that case it wouldn't be flagged by vale anyway.

Reply:
I don't think this works, at least it doesn't seem to in my editor

```suggestion
docker/
```

Reply:
LOL what's your suggested change above? I don't see what it ... changed 🥹